exec/jdbc/src/test/java/org/apache/drill/jdbc/PreparedStatementTest.java

```diff
@@ -462,4 +618,25 @@ public void testParamSettingWhenUnsupportedTypeSaysUnsupported() throws SQLExcep
     }
   }
 
+
+  // Sets the SystemMaxRows option
+  private void setSystemMaxRows(int sysValueToSet) throws SQLException {
+    // Acquire lock if Non-Zero Value (i.e. a test is in progress)
+    if (sysValueToSet != 0) {
+      maxRowsSysOptionLock.acquireUninterruptibly();
+    }
+    // Setting the value
+    try (Statement stmt = connection.createStatement()) {
+      stmt.executeQuery(ALTER_SYS_OPTIONS_MAX_ROWS_LIMIT_X + sysValueToSet);
+      ResultSet rs = stmt.getResultSet();
```

Comment:
Is it required to obtain a result set and iterate thru it?

Reply:
No. If the query fails, you won't get the resultset object.

Reply:
Ok, thanks for the clarification.